sycl/test-e2e/Graph/Update/dyn_cgf_dyn_local_accessor.cpp

```diff
@@ -0,0 +1,88 @@
+// RUN: %{build} -o %t.out
+// RUN: %{run} %t.out
+// Extra run to check for leaks in Level Zero using UR_L0_LEAKS_DEBUG
+// RUN: %if level_zero %{env SYCL_PI_LEVEL_ZERO_USE_IMMEDIATE_COMMANDLISTS=0 %{l0_leak_check} %{run} %t.out 2>&1 | FileCheck %s --implicit-check-not=LEAK %}
+// Extra run to check for immediate-command-list in Level Zero
+// RUN: %if level_zero %{env SYCL_PI_LEVEL_ZERO_USE_IMMEDIATE_COMMANDLISTS=1 %{l0_leak_check} %{run} %t.out 2>&1 | FileCheck %s --implicit-check-not=LEAK %}
+
+// XFAIL: level_zero
+// XFAIL-TRACKER: OFNAAO-422
```

Comment:
Think we can remove this XFAIL from these tests now, see https://github.com/intel/llvm/pull/16798

Reply:
I've just checked out this git branch to test, and confirm we can remove the XFAIL